### PR TITLE
Boot pool table refactor

### PR DESCRIPTION
**Pull request by joshuarule**

### CODE REVIEW

**Comment by joshuarule** [comment]
requires merge of https://github.com/freenas/blue-shark/pull/14

File: ui/inspectors/system-section.reel/boot-pool.reel/boot-environment-row.reel/boot-environment-row.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @module ui/boot-environment-row.reel
+ */
+var Component = require("montage/ui/component").Component;
+
+/**
+ * @class BootEnvironmentRow
+ * @extends Component
+ */
+exports.BootEnvironmentRow = Component.specialize(/** @lends BootEnvironmentRow# */ {
+    constructor: {
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/boot-pool.reel/boot-environment-row.reel/boot-environment-row.js]: You should remove this method, it doesn't do anything, it is just some noise added by `minit`.

File: ui/inspectors/system-section.reel/boot-pool.reel/boot-pool-action.reel/_boot-pool-action.css [hunk 2]
```diff
@@ -1,14 +1,51 @@
 .BootPoolAction {
-    display: flex;
-    flex-wrap: wrap;
-    width: 11em;
-    margin-bottom: -.25em;
+    /* display: flex; */
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/boot-pool.reel/boot-pool-action.reel/_boot-pool-action.css]: You should remove those lines completely.

File: ui/inspectors/system-section.reel/boot-pool.reel/boot-pool-action.reel/_boot-pool-action.css [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+    display: none;
 }
 
+
+
 .BootPoolAction > * {
-    flex: 1;
-    margin-bottom: .25em !important;
-    margin-right: .25em !important;
+    /* flex: 1; */
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/boot-pool.reel/boot-pool-action.reel/_boot-pool-action.css]: Same here.

File: ui/inspectors/system-section.reel/boot-pool.reel/boot-pool.js [hunk 4]
```diff
@@ -62,85 +78,57 @@ exports.BootPool = Component.specialize(/** @lends BootPool# */ {
...
+    handleAction: {
         value: function (event) {
-            var bootEnvironment = this._findBootEnvironmentFromActionEvent(event);
 
-            if (bootEnvironment) {
-                return this._performAction(
-                    event,
-                    this._bootEnvironmentService.persistBootEnvironmentRenaming(bootEnvironment)
-                );
+            if(event.detail && event.detail.get('eventName') == 'textValueChanged') {
```
Comment (pchaussalet) [inline ui/inspectors/system-section.reel/boot-pool.reel/boot-pool.js:100]: @thibaultzanini Any opinion on this ?

**Review by pchaussalet — CHANGES_REQUESTED** [review]

**Review by pchaussalet — APPROVED** [review]

**Comment by pchaussalet** [comment]
LGTM

**Comment by joshuarule** [comment]
this was merged manually

**Pull request closed without merging**